vms/proposervm/block.go

```diff
@@ -24,6 +24,11 @@ import (
 const (
 	// allowable block issuance in the future
 	maxSkew = 10 * time.Second
+
+	// unspecifiedSlotIndex defines the default value being used for a proposal slot index, prior of being populated
+	// by the actual slot index. Given that the slot index is zero base, we would want to use an "invalid" value as
+	// non-zero.
+	unspecifiedSlotIndex = ^uint64(0)
```

Comment:
Optional: Might be a bit more clear on what this value is (and why we never expect a slot to be equal to this)
```suggestion
	unspecifiedSlotIndex = math.MaxUint64
```